app/src/main/java/com/sobolevkir/playlistmaker/App.kt

```diff
@@ -0,0 +1,29 @@
+package com.sobolevkir.playlistmaker
+
+import android.app.Application
+import androidx.appcompat.app.AppCompatDelegate
+import com.sobolevkir.playlistmaker.PrefsManager.IS_DARK_THEME_ON
+
+class App: Application() {
+
+    private var isDarkThemeOn = false
+
+    override fun onCreate() {
+        super.onCreate()
+        PrefsManager.init(this.applicationContext)
+        isDarkThemeOn = PrefsManager.read(IS_DARK_THEME_ON, false)
```

Comment:
```suggestion
        isDarkTheme = PrefsManager.read(IS_DARK_THEME_ON, false)
```
Без "on" в конце и короче и понятнее